inc/setup/cachify.apc.htaccess.php

```diff
@@ -29,8 +29,7 @@
 		</tr>
 	</table>
 
-	<div style="background:#fff;border:1px solid #ccc;padding:10px 20px">
-		<pre style="white-space: pre-wrap">
+	<textarea rows="5" class="large-text code" name="code" id="cachify-code" readonly>
```

Comment:
Is there a specific reason to specify an ID instead of an additional class?

Reply:
readonly is forcing it to gray. so we have to use !important or id

Reply:
We have to use a selector that is more precise than ´textarea[readonly]`. `.cachify-code` is not.

`#cachify-code` is such selector, but `textarea.cachify-code` also.